framework/rbac/DbManager.php

```diff
@@ -600,6 +608,10 @@ public function getAssignment($roleName, $userId)
         if (empty($userId)) {
             return null;
         }
+        if ($this->cacheAssignments) {
```

Comment:
Current code would result in extra SQL query from `getAssignments` in case these aren't in the cache yet.